src/dotnet/APIView/APIViewWeb/APIViewWeb.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="bin\**" />
```

Comment:
Hm, what is this for?

Reply:
Ted and I had to resolve loading issues caused by package conflicts, and when we created a new solution to transfer the existing code over some changes were created in the solution and project files. Honestly, I'm not too sure what the added code does, but it's fixed the issue.

Reply:
Let's try to remove them, I'll help you solve the issue if it comes back but `git clean` might do it too.

Reply:
Yeah, there was weirdness.  Now that you've got the changes staged as part of this PR, it shouldn't be problematic to try yanking these from the `.csproj` files and seeing what happens.